[WIP] P11 develop 2

agruzinov opened the pull request:
Small changes to the P11 HO after testing.

marcus-oscarsson committed:
[Exporter] - Handling runScript for exporter protocol

marcus-oscarsson committed:
[Collect] - Passing number of images and removed number of passes (no longer used)

marcus-oscarsson committed:
[WF] - Setting characterization result via XMLPRC instead of reading from file system

marcus-oscarsson committed:
[Cleanup] - Removed commented code

marcus-oscarsson committed:
[ESRF HO] - Improved error handling

marcus-oscarsson committed:
[Collect] - Data collection speed up

marcus-oscarsson committed:
[Collect] - Passing correct parameters after signature change

marcus-oscarsson committed:
[MD] - Converting value to corresponding enum on value update event

marcus-oscarsson committed:
[ESRF HO] - Added missing files

marcus-oscarsson committed:
[ESRF HO] - Starting run number counting at 1

marcus-oscarsson committed:
[MD] - Added abort method

marcus-oscarsson committed:
[Detector] - Updating detector status when doing stop acquisition

marcus-oscarsson committed:
[Detector] - Configurable trigger mode for Pilatus detector

marcus-oscarsson committed:
[MD] - Added new scans for time resolved experiments

marcus-oscarsson committed:
[MD] - Updated scans after change on MD

marcus-oscarsson committed:
[Collect] - Fixed frame counting issue

marcus-oscarsson committed:
[WF]  - Fixing imports, making sure we can import queue_model_objects standalone

marcus-oscarsson committed:
[Lint] - Added missing import

marcus-oscarsson committed:
[Detector] - Make sure detector is ready before preparing

marcus-oscarsson committed:
[SC] - Don't change to transfer phase in unload

marcus-oscarsson committed:
[ESRF HO]: Made handling of beam size more robust

marcus-oscarsson committed:
[Mockup] - Fixed set value

marcus-oscarsson committed:
[Beamline Actions] - Only emit commandReplyArrived if action is running

marcus-oscarsson committed:
[Sample Changer] - Emit gripperChanged

marcus-oscarsson committed:
[skip ci] Bumped minor version

Commit:
Fixed missing var

Commit:
Removed unused commented code

Commit:
Fixed centring

Commit:
Debug of charactarisation part

fabcor-maxiv commented on pyproject.toml:
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "mxcubecore"
-version = "1.76.0"
+version = "1.77.0"
```
As far as I understood the process, that should not be done from within a pull request.

```suggestion
version = "1.76.0"
```

marcus-oscarsson replied:
Its indeed strange that the PR includes a previous commit, from the CI (#4f6c5d9)

agruzinov replied:
That happens if I get the latest updated from the upstream develop to agruzinov/develop through github web interface.

marcus-oscarsson committed:
Lint

marcus-oscarsson committed:
Lint

marcus-oscarsson committed:
Lint

marcus-oscarsson committed:
Update create-tag.yml

marcus-oscarsson committed:
Fixed typo

marcus-oscarsson committed:
Added missing copy import and using deepcopy

marcus-oscarsson committed:
[skip ci] Bumped minor version

Commit:
ISPyB mock: drop ldap server code

Remove the code that 'deals' with authServerType and ldapServer
ISPyBClientMockup properties.

The LDAP mode is not really used in the code. The object does not
change behaviour when authServerType is changed. Let's drop LDAP
bits, as it creates unnecessary complication when using MxCube
with mock hwobjs.

marcus-oscarsson committed:
[skip ci] Bumped minor version

agruzinov committed:
Merge branch 'mxcube:develop' into p11_develop_2

agruzinov commented:
Is there a cleaner way to incorporate all the changes and having a fork without having all the commits that happened in between after rebase? 

For tnow it is like this:

There is a branch of core/qt under mu user.
Than:

-Synchronise all the changes that happened in between with the develop branch through the web interface. 
Than on the develop maschine:
git checkout develop
git pull (with rebase)

than:

git checkout p11_develop_2
git rebase develop
git push

Rhetorical question: what I'm doing wrong? :)

Or it is easier just to to work on the main repo than?

Any comments will be appreciated? Sorry for the basic question, but I think I've lost the understanding of the git behaviour.

Cheers!



git branch p11_develop_2

agruzinov commented:
Resolved through another PR#905. Closing.
